native/src/components/base/InputSection.tsx

```diff
@@ -30,50 +31,65 @@ const Title = styled(ThemedText)`
   text-align: left;
 `
 
-const Input = styled.TextInput<{ numberOfLines: number }>`
+const Input = styled.TextInput<{ numberOfLines: number; invalid: boolean }>`
   border-width: 1px;
-  border-color: ${props => props.theme.colors.themeColor};
-  text-align-vertical: top;
+  border-color: ${props => (props.invalid ? props.theme.colors.invalidInput : props.theme.colors.themeColor)};
```

Comment:
So in malte the default color is dead read? Yikes! But nothing to do now I guess...

Reply:
Good point, I guess I will just make the borders grey (text decoration color).

Reply:
Will also affect integreat incl. the feedback component then though.

Reply:
In web it is already black. So I don't think this should be an issue ;)